Source/Core/Core/Core.cpp

```diff
@@ -285,7 +283,7 @@ void Stop()  // - Hammertime!
 		g_video_backend->Video_ExitLoop();
 	}
 #if defined(__LIBUSB__) || defined(_WIN32)
```

Comment:
Do you still need this define? The include is outside the defines too (unless I missed some higher-level define in the GitHub interface)